scripts/linter/collect-docs.js

```diff
@@ -0,0 +1,32 @@
+const fsPromises = require("fs").promises;
+const path = require("path");
+
+/**
+ * Walks the content directory tree (`start`) yielding Markdown files.
+ */
+async function* walkContent(start = "content") {
+    const files = await fsPromises.readdir(start, { withFileTypes: true });
+
+    for (const f of files) {
+        if (f.name.endsWith(".md")) {
+            yield path.join(start, f.name);
+        } else if (f.isDirectory()) {
```

Comment:
The order is wrong. If there's a folder called `content/stuff/foo.md/` it'd yield that instead of digging deeper into it.

Reply:
Good catch. Fixed.